lib/rubocop/cop/mixin/target_solidus_version.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for checking target solidus version.
+    module TargetSolidusVersion
+      DEFAULT_SOLIDUS_VERSION = '3.0'
+      RUBOCOP_SOLIDUS_HOME = File.realpath(File.join(File.dirname(__FILE__), '..', '..', '..', '..')).freeze
+      DOTFILE = '.rubocop.yml'
+
+      def minimum_supported_solidus_version?(version)
```

Comment:
Function to determine if `target_solidus_version` is higher than minimum required version of any particular cop